README.md

```diff
@@ -61,36 +34,49 @@ curl -OL https://github.com/Kitware/nrtk-explorer/archive/refs/heads/main.zip
 
 Inside the application source code top directory, install it with:
 
-```
+```bash
 pip install -e .
```

Comment:
Use package name instead. The `-e .` is more for dev setup which is covered below.

Reply:
I think the `-e .` line was intended as a second step to  `curl` style source code "checkout."  

I just delete those 2 steps.